Makefile

```diff
@@ -51,3 +51,8 @@ coverage:
 	mkdir -p $(COVERAGE_DIR)
 	uv run python -m coverage report -m > $(COVERAGE_DIR)/coverage.txt
 	uv run python -m coverage html --directory=$(COVERAGE_DIR)/html/
+
+
+.PHONY: train-canonical-models
+train-canonical-models:
+	spd-run --local --create_snapshot False --create_report False --log-format terse
```

Comment:
create_snapshot and create_report don't need to be specified when running locally. Also, it's nice if people share reports for canonical runs wherever they're needed. Given that, I'm not even sure it's worth putting in the makfile. We can just tell the user in the docstring of the registry (and README) to run `spd-run` or `spd-run --local` for the canonical runs.